script/releases/v0.5.1-rewardsv2/1-eoa.s.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.12;
+
+import {EOADeployer} from "zeus-templates/templates/EOADeployer.sol";
+import {RewardsCoordinator} from "src/contracts/core/RewardsCoordinator.sol";
+import {IDelegationManager} from "src/contracts/interfaces/IDelegationManager.sol";
+import {DelegationManager} from "src/contracts/core/DelegationManager.sol";
+import {StrategyManager} from "src/contracts/core/StrategyManager.sol";
+import {EigenLabsUpgrade} from "../EigenLabsUpgrade.s.sol";
+import {Test, console} from "forge-std/Test.sol";
+import {IPauserRegistry} from "src/contracts/interfaces/IPauserRegistry.sol";
+
+contract Deploy is EOADeployer {
+    using EigenLabsUpgrade for *;
+
+    function _runAsEOA() internal override {
+        vm.startBroadcast();
+        
+        deploySingleton(
+            address(
+                new RewardsCoordinator(
+                    IDelegationManager(zDeployedProxy(type(DelegationManager).name)),
+                    StrategyManager(zDeployedProxy(type(StrategyManager).name)),
+                    zUint32("REWARDS_COORDINATOR_CALCULATION_INTERVAL_SECONDS"),
+                    zUint32("REWARDS_COORDINATOR_MAX_REWARDS_DURATION"),
+                    zUint32("REWARDS_COORDINATOR_MAX_RETROACTIVE_LENGTH"),
+                    zUint32("REWARDS_COORDINATOR_MAX_FUTURE_LENGTH"),
+                    zUint32("REWARDS_COORDINATOR_GENESIS_REWARDS_TIMESTAMP")
+                )
+            ),
+            this.impl(type(RewardsCoordinator).name)
+        );
+
+        vm.stopBroadcast();
+    }
+
+    function testDeploy() public {
+        _runAsEOA();
+        Deployment[] memory deploys = deploys();
+
+        // sanity check that zDeployedImpl is returning our deployment.
+        assertEq(deploys[0].deployedTo, zDeployedImpl(type(RewardsCoordinator).name));
+
+        RewardsCoordinator rewards = RewardsCoordinator(zDeployedImpl(type(RewardsCoordinator).name));
+
+        address ms = this._executorMultisig();
+        IPauserRegistry pauserRegistry = IPauserRegistry(this._pauserRegistry());
+        uint64 initPausedStatus = zUint64("REWARDS_COORDINATOR_INIT_PAUSED_STATUS");
+        address coordinatorUpdater = zAddress("REWARDS_COORDINATOR_UPDATER");
+        uint32 activationDelay = zUint32("REWARDS_COORDINATOR_ACTIVATION_DELAY");
+
+        vm.expectRevert();
+        rewards.initialize(
+            ms,
+            pauserRegistry,
+            initPausedStatus,
+            coordinatorUpdater,
+            activationDelay,
+            1000
```

Comment:
Any reason to not use the constant in the metadata repo like the other variables above for the default split bips?